src/tools/dart-create.md

```diff
@@ -0,0 +1,82 @@
+---
+title: dart create
+description: Command-line tool for creating Dart projects.
+toc: false
+---
+
+The `dart create` command creates a Dart project,
+using one of several supported templates.
+The same functionality is available in IDEs.
+
+{% include tools/dart-tool-note.md %}
+
+When you run `dart create`, it first creates a directory with the project files. 
+Then it gets package dependencies (unless you specify the `--no-pub` flag.) 
+
+Here's an example of using `dart create` to create a directory named `my_cli` 
+that contains a simple console app (the default template):
+
+```terminal
+$ dart create my_cli
+```
+
+To use a different template, such as `web-simple`, add a template argument:
+
+```terminal
+$ dart create -t web-simple my_web_app
+```
+
+The following table shows the templates you can use:
+
+|------------------+------------------------------------------------------|
+| Template         | Description                                          |
+|------------------|------------------------------------------------------|
+| `console-simple` | A simple command-line application.                   |
```

Comment:
Google style is to [use _app_ instead of _application_](https://developers.google.com/style/word-list#app) except where you want to convey complexity. Also, I wanted to make clear that console-simple is the default.

```suggestion
| `console-simple` | A simple command-line app (the default template).                   |
```